### ISSUE TITLE
Chromium browsers issue with mouse tracking/reading

### ISSUE THREAD

**Issue by West-Palm-Beef** [nvaccess/nvda#17613]
### Steps to reproduce:
Either be in Google Chrome, OperaGX, or other chromium based browsers, with or without NVDA started up.
Make sure to have the mouse tracking feature on, so that when a user hovers over text or other elements, it will detect and read them out loud, and/or highlight depending on the visual settings activated.

### Actual behavior:
When trying to enter the browser with just the mouse, there will either be a very basic announcement of an element NVDA decides to place the virtual cursor on; no announcement; or it will say, "pane unavailable", and user is unable to have elements read out to them.

### Expected behavior:
When user enters browsing window, the mouse can hover over text elements, links, buttons, images, etc., and have them read aloud to the user as they try to read with the mouse.

#### NVDA installed/portable/running from source:
Directly on drive

#### NVDA version:
Latest as of 1/12/2025

#### Name and version of other software in use when reproducing the issue:
Windows magnifier for zooming in on text or images.
Google Chrome, OperGX, or other chromium-based browser (as is to my understanding of how these things work, I am definitely not a browser dev...)

#### Other information about your system:
AMD processor, PNG Nvidia RTX 2070Ti, and 128GB of RAM

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
N/A

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
No

**Comment by West-Palm-Beef** [comment]
Please note, if there is a lack of technical phrasing or something I am not using, it is because I am not a professional dev, I am just kind of a beginner to this space, GH in particular.

I may or may not have understood some things correctly, and am going to check  back here frequently, so if there are any questions, I will make sure to do my best to answer them! :)

**Comment by hwf1324** [comment]
Hi,

I don't quite understand your description, it seems a bit vague.

Mouse tracking works fine on NVDA alpha 35001 and Chrome version 131.0.6778.265 (official) (64-bit), reporting text, links, etc. in web pages.

If you can please clarify the Steps to reproduce and Actual behavior etc.

Have you tried the COM Registration Fixing Tool under the Tools menu?

Thanks!

**Comment by gerald-hartig** [comment]
Duplicate of #11845
